### PR TITLE
[MM-36428][MM-36434][MM-36435] Keyboard navigation and menu updates for new tab/dropdown layout

### PR DESCRIPTION
**Pull request by devinbinnie**
#### Summary
This PR contains all the keyboard navigation and menu updates, so all of the menu items and keyboard shortcuts should look and work as expected with the new layout.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-36428
https://mattermost.atlassian.net/browse/MM-36434
https://mattermost.atlassian.net/browse/MM-36435
https://mattermost.atlassian.net/browse/MM-37863

#### Release Note
```release-note
Updated some keyboard shortcuts and menu items to work with the new desktop app layout
```

### CODE REVIEW

**Review by matthewbirtch — CHANGES_REQUESTED** [review]
@devinbinnie as we discussed, shortcuts will need to be as follows:

**Mac**
- `CMD` `CTRL` `S` - Open Servers Dropdown
- `CMD` `CTRL` `N` - Open Serve N in the list

**Windows**
- `CTRL` `SHIFT` `S` - Open Servers Dropdown
- `CTRL` `SHIFT` `N` - Open Serve N in the list

**Review by matthewbirtch — APPROVED** [review]
Tested the shortcuts on both mac and windows and verified they work as expected. Nice work @devinbinnie!

**Review by deanwhillier — APPROVED** [review]
LGTM, though I'm a little out of touch with the new architecture. 😛

File: src/main/menus/app.ts [hunk 1]
```diff
@@ -206,22 +206,35 @@ function createTemplate(config: Config) {
...
                     WindowManager.switchServer(team.name);
                 },
-            };
-        }), separatorItem, {
-            label: 'Select Next Server',
+            });
+            if (WindowManager.getCurrentTeamName() === team.name) {
+                team.tabs.slice(0, 9).sort((teamA, teamB) => teamA.order - teamB.order).forEach((tab, i) => {
+                    items.push({
+                        label: `    ${tab.name}`, // TODO
```
Comment (BenCookie95) [inline src/main/menus/app.ts:220]: What is the TODO here?

**Review by BenCookie95 — APPROVED** [review]

File: src/main/menus/app.ts (hunk 1 above)
Comment (devinbinnie) [reply src/main/menus/app.ts:220]: This should change to a display name once https://github.com/mattermost/desktop/pull/1680 is merged.

File: .circleci/config.yml [hunk 2]
```diff
@@ -449,7 +449,7 @@ workflows:
             branches:
               only:
                 - /^release-\d+(\.\d+){1,2}(-rc.*)?/
-                - pull/1691
+                - pull/1695
```
[the remark this answers is not in the document]
Comment (devinbinnie) [reply .circleci/config.yml:452]: Yeah, easy way to get builds made. I'm working on a better way :P

**Pull request merged**